pkg/operator/controller/externaldns/deployment_test.go

```diff
@@ -58,6 +58,7 @@ var (
 )
 
 func TestDesiredExternalDNSDeployment(t *testing.T) {
+	secretHash := "f17bceb5a060e33473c68229903ef5c517d9a172"
```

Comment:
nit: a suggestion would be to make this a constant, as it is used in several places.

Reply:
Resolved here 4725c696ebfa116c8147cc3bd98362e8bc76c64f